doc/builddoc.rst

```diff
@@ -1,28 +0,0 @@
-.. _build_doc:
```

Comment:
Are we removing this from our documentation? Or moving it elsewhere?

Reply:
The documentation is build by Travis and has this own make rule, then for me it don't needs a specific (and probably unmaintained) documentation...

Reply:
That's fine for me, but I think then you need to remove references to "build_doc", too

Reply:
For me it's done, do you see an other one?